const loginUser = (req,res) =>{
    try{

    }catch(error){
        
    }
}